README.md

```diff
@@ -86,161 +86,204 @@ session-service$ export set SERVER_PORT=8090; mvn package -Dpackaging.type=jar &
 
 ### Create
```

Comment:
There is a tool called swagger that we've used to document the cbioportal webservice using a library called SpringFox.  We should integrate this at some point.  I can give you more information.